app/models/organization.js

```diff
@@ -11,4 +11,5 @@ export default Owner.extend({
   iconLargeUrl: attr(),
   projects: DS.hasMany('project', { async: true }),
   members: DS.hasMany('user', { async: true }),
+  organizationMemberships: DS.hasMany('organization-membership', { async: true }),
```

Comment:
Do you mean the `async` part? Isn't `true` the default for that?

I agree with going the async approach in most cases either way. While there are more required endpoints on the API that way, as well as more network requests, I believe it simplifies things on both ends overall.